docs/csharp/index.yml

```diff
@@ -101,17 +157,13 @@ landingContent:
           - text: Version compatibility
             url: whats-new/version-update-considerations.md
 
-  - title: "C# language reference"
```

Comment:
I'd like to have this card brought back. There's a lot of traffic that goes to the reference and spec sections. It can be near the end, below the fold, because the readers going here generally know where they want to go.